projects/plugins/jetpack/modules/likes/style.css

```
@@ -212,8 +212,14 @@ div.sharedaddy.sd-like-enabled .sd-like .post-likes-widget {
 	display: block !important;
 }
 
-
 /* Hide G+ title */
 .sd-gplus .sd-title {
 	display: none;
+	
+}
+
+@media print {
+	.sharedaddy.sd-sharing-enabled{
+		display: none;
+	}
```

```suggestion
}

@media print {
	.sharedaddy.sd-sharing-enabled {
		display: none;
	}
```
I think you added some unintended whitespace. Looks good otherwise!